util/src/main/java/org/datacommons/util/PlaceSeriesSummary.java

```diff
@@ -0,0 +1,109 @@
+package org.datacommons.util;
+
+import com.google.common.hash.Hasher;
+import com.google.common.hash.Hashing;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.TreeMap;
+import org.datacommons.proto.Debug.DataPoint;
+import org.datacommons.proto.Debug.DataPoint.DataValue;
+import org.datacommons.proto.Debug.StatValidationResult;
+import org.datacommons.proto.Mcf.McfGraph;
+import org.datacommons.util.SummaryReportGenerator.StatVarSummary;
+
+// An object to hold all the series information for all the statistical variables for a place.
+public class PlaceSeriesSummary {
+  public static class SeriesSummary {
+    StatValidationResult.Builder validationResult;
+    TreeMap<String, DataPoint> timeSeries;
```

Comment:
nit: comment why TreeMap (presumably to keep keys sorted?)

Reply:
done